Tools/NTupleReader.h

```diff
@@ -125,7 +125,7 @@ class NTupleReader
     class array_deleter : public vec_deleter<T>
     {
     public:
-        void deletePtr(void* ptr) {}
+        void deletePtr(void*) {}
```

Comment:
why?

Reply:
See the above reason.

Reply:
Whoops this is different than the above reason. 
"ptr" is not used in the function so there is no reason to state it. All this functions needs is the type.

Reply:
this solution is fine, but you can also fo the following to mark function parameters unused

```
int f(void* p)
{
    (void)p;
}